python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -221,8 +223,10 @@ def fromDict(cls, dictionary):
         calib.shape = (dictionary['metadata'].get('KERNEL_DX', 0),
                        dictionary['metadata'].get('KERNEL_DY', 0))
 
-        calib.means = {amp: np.array(dictionary['means'][amp]) for amp in dictionary['means']}
-        calib.variances = {amp: np.array(dictionary['variances'][amp]) for amp in dictionary['variances']}
+        calib.expIdMask = {amp: np.array(dictionary['expIdMask'][amp]) for amp in dictionary['rawXcorrs']}
```

Comment:
Can these iterate over their own amplifier sets (`for amp in dictionary['expIdMask']}` etc)?  There's nothing wrong as it is, but if these have different lengths, I'd like it to give a clear message about what entry doesn't have a full set of amps.